modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1168,6 +1183,11 @@ GrainTracker::remapGrains()
       _console << "Swaps complete" << std::endl;
   }
 }
+void
+GrainTracker::remapGrainsMerge()
+{
+  _console << "This function needs to be defined in the derived class\n" << std::endl;
```

Comment:
You probably don't want to print this message out here. This won't make any sense for existing use of the GrainTracker class. Just leave the method empty.

Reply:
Ok, redundant code has been removed.